fh_immuta_utils/tagging.py

```diff
@@ -19,6 +17,13 @@
 LOGGER = logging.getLogger(__name__)
 
 
+# Tags created by Immuta by default
+IMMUTA_SPECIAL_TAGS = {
```

Comment:
Do we know how often these are changed by Snowflake?

Reply:
these are created by Immuta. It should change very infrequently if ever.